docs/3.tutorials/examples/count-near.md

```diff
@@ -192,7 +188,7 @@ near create-account <accountId> --useFaucet
 
 # Deploy the contract
 cd contract-rs
-./build.sh
+cargo build
 near deploy <accountId> ./target/wasm32-unknown-unknown/release/counter.wasm
```

Comment:
I guess it's better to use `cargo near deploy <accountId>` command since we follow the new approach with the whole `cargo ...` thing